src/TSHelper.ts

```diff
@@ -419,10 +419,25 @@ export function getCustomNodeDirectives(node: ts.Node): Map<DecoratorKind, Decor
 }
 
 export function getCustomFileDirectives(file: ts.SourceFile): Map<DecoratorKind, Decorator> {
+    const directivesMap = new Map<DecoratorKind, Decorator>();
+
     if (file.statements.length > 0) {
-        return getCustomNodeDirectives(file.statements[0]);
+        // Manually collect jsDoc because `getJSDocTags` includes tags only from closest comment
+        const jsDoc = file.statements[0].jsDoc;
```

Comment:
Is it possible for a file to have no statements?

Reply:
Yes, for empty file `SourceFile.statements` is an empty array, that's why there is a length check above.

Reply:
Sorry - I completely missed that